litellm/main.py

```diff
@@ -38,7 +53,8 @@
 
 ####### COMPLETION ENDPOINTS ################
 #############################################
-def completion(model, messages, azure=False):
+@func_set_timeout(10, allowOverride=True)
```

Comment:
@krrishdholakia from PR #12 - I don't think we should default all users with a 10s timeout

Reply:
from @krrishdholakia: Reason for token limit - Anthropic requires token limits. Their defaults are 300.

Reply:
**Order for PRs pushing to main:**
 - @krrishdholakia pushing latest version with litellm client 
 - @zakhar-kogan adding poetry 
 - @zakhar-kogan this current PR #14

Reply:
@krrishdholakia set timeout for 60s